src/fn/fn-form.scss

```diff
@@ -0,0 +1,65 @@
+@import "./fn-settings";
+
+$block: #{$fn-namespace}-form;
+
+.#{$block} {
+  @include fn-reset();
+
+  &-item {
```

Comment:
You wrote `fn-form__item` in description but here it's `fn-form-item`, which one is correct?

Reply:
good catch, should be `fn-form__item`